pastis/plotting.py

```diff
@@ -994,3 +995,130 @@ def natural_keys(text):
     https://stackoverflow.com/questions/5967500/how-to-correctly-sort-a-string-with-a-number-inside)
     """
     return [atoi(c) for c in re.split(r'(\d+)', text)]
+
+
+def plot_thermal_mus(mus, nmodes, nsegments, c0, out_dir, save=False):
+    """
+    Generates modal constraints plot for individual segment.
+
+    Parameters
+    ----------
+    mus : ndarray
+        list of standard deviations for each segment
+    nmodes : int
+        number of thermal modes
+    nsegments :  int
+        number of segments
+    c0 : scalar
+        target contrast
+    out_dir : str
+        path to save the plot
+    save : bool
+        whether to save the plot
+    """
+    harris_coeffs_table = np.zeros([nmodes, nsegments])
+    for qq in range(nmodes):
+        for kk in range(nsegments):
+            harris_coeffs_table[qq, kk] = mus[qq + (kk) * nmodes]
+
+    plt.figure(figsize=(10, 10))
+    plt.title("Modal constraints to achieve a dark hole contrast of "r"$10^{%d}$" % np.log10(c0), fontsize=20)
+    plt.ylabel("Weight per segment (in units of pm)", fontsize=15)
+    plt.xlabel("Segment Number", fontsize=20)
```

Comment:
Are we still doing segment numbers here or should they be mode numbers?

Reply:
Yes it is segment number. We want this very much.